### DISCUSSION TITLE
Error running 'pscale shell...' commands

### DISCUSSION THREAD

**Discussion by finicky-pet** [planetscale/discussion#68] [Q&A] [answered]
I get this error on Windows 10 when running `'pscale shell...'` commands.` Error: couldn't find the 'mysql' command-line tool required to run this command.`

I've installed, and resinstall the mysql package, added to system variables, etc
Can someone point me in the right direction please?

**Comment by jgreet** [comment, marked as answer]
​
`pscale` is looking for the `mysql.exe` command-line client. 

If you [installed MySQL Server 8.0](https://dev.mysql.com/doc/refman/8.0/en/windows-installation.html) in the default location, you can likely find it here:  `C:\Program Files\MySQL\MySQL Server 8.0\bin\mysql.exe`.

Once you've added the directory containing mysql.exe to your shell's PATH, you should be able to run both `mysql` and `pscale shell` on the command line.
​
In PowerShell, you can add the directory to your PATH with:

```powershell
$env:path += ";C:\Program Files\MySQL\MySQL Server 8.0\bin"
```
